docs/Development.md

```diff
@@ -107,10 +107,17 @@ When a unit test generates a PDF, it is recommended to use the `assert_pdf_equal
 utility function in order to validate the output.
 It relies on the very handy [qpdf](https://github.com/qpdf/qpdf) CLI program
 to generate a PDF that is easy to compare: annotated, strictly formatted,
-whith uncompressed internal streams.
+with uncompressed internal streams.
 You will need to have its binary in your `$PATH`,
 otherwise `assert_pdf_equal` will fall back to hash-based comparison.
-In order to generate a "reference" PDF file, simply call `assert_pdf_equal` once with `generate=True`.
+
+All generated PDF files (including those processed by `qpdf`) will be stored in
+`/tmp/pytest-of-USERNAME/pytest-current/NAME_OF_TEST/`. By default, three
+last test runs will be saved and then automatically deleted, so you can
```

Comment:
Thank you for the documentation update!

How does this logic of "three last test runs" is implemented?

Also, given that the temporary PDF file names do not include the test name,
it is necessary to execute failing tests on their own to be able to inspect their corresponding PDFs.
Hence, could you please add a mention of `pytest -k $pattern` in this paragraph?

Reply:
`/tmp/pytest-of-$USER/pytest-current` is a symbolical link to `/tmp/pytest-of-$USER/pytest-0` (with `0`, `1`, `2`, `3`, etc. for each new test run, always the last three runs, older will be deleted by `pytest`). This number is configurable, but the default is three, which is okay in most cases.

Each test function gets its own subdirectory `NAME_OF_TEST`, so at the end you'll have wide directory structure and in (almost) every subdirectory you'll find files `actual.pdf`, `actual_qpdf.pdf` and `expected_qpdf.pdf`. 

One example complete path of such a file is: `/tmp/pytest-of-miro/pytest-0/test_rotation0/actual.pdf`

So there's no need to specifically document the `pytest -k $pattern` because all the files are there. However, if you feel I should reword the documentation, please let me know.

Reply:
Oh and if the test function is parametrized, each parameter will get its own comprehensive subdirectory name.

Just an extra idea: we could automatically copy the expected test file to the respective directory as `expected.pdf`, which would make the content of those directory complete. What do you think?

Reply:
Thank you for the detailed explanation!

I did not about this behaviour of `pytest`, it's really handy :)

I do not think it's necessary to copy the expected file.

Merging this now, thanks for your contribution!